themes/bootstrap3/templates/combined/stack-grid.phtml

```diff
@@ -1,5 +1,5 @@
 <div class="combined-search-container grid"
-  style="grid-template-columns: repeat(<?=$this->escapeHtmlAttr($columns)?>, 1fr)"
+  style="grid-template-columns: repeat(<?=$this->escapeHtmlAttr($columns)?>, minmax(0, 1fr))"
```

Comment:
I'm really not clear why setting an explicit minimum has the effect of enforcing the expected maximum.  Maybe someone else can parse MDN better.  https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns

Reply:
It looks like minmax is overriding the overflow consideration. It is strictly setting the width rather than let them grow. It seems to be the better option.